src/openlayers-element.ts

```diff
@@ -124,6 +126,18 @@ export class OpenLayersElement extends LitElement {
       });
       this.geolocation.setTracking(true);
       new GeolocationMarker(map, this.geolocation);
+      map.addControl(new LoaderBoxControl("Chargement des données GPS"));
+      const waitGeolocation = setInterval(() => {
+        if (this.geolocation?.getPosition()) {
+          clearInterval(waitGeolocation);
```

Comment:
not enough to clear it when geolocation received, you should hanle case of errors (user disallow position, gps errors, etc.). Also probably needed to call clearIntervall in disconnectedCallback if not cleared yet

Reply:
Full refactoring of this part done.